lib/Qi.pm

```diff
@@ -10,7 +14,9 @@ sub startup {
   my $config = $self->plugin('Config');
 
   #Model
-   $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(sql => sub{ state $sql = SQL::Abstract->new});
```

Comment:
Interesting! I assumed it would come down to a tossup beween optimization and readability. Glad to see the argument for design optimization. Super interesting!